src/GraphBLAS-sharp/CSRMatrix.fs

```diff
@@ -0,0 +1,113 @@
+namespace GraphBLAS.FSharp
```

Comment:
Why CSR matrix and related stuff are also added in this PR? Looks like it is a reason of the merge conflict.

Reply:
I guess you should pull the current master to your branch first.

Reply:
That stuff is added because it uses entity that I changed (mask). I had to correct this files to avoid problems with them. But to be honest I can't understand why these corrections are recognized as conflicts, I'll try to pull master